pyproject.toml

```diff
@@ -48,6 +48,7 @@ dev = [
     "pytest~=8.2.2",
     "pytest-cov~=5.0.0",
     "pytest-mock~=3.14.0",
+    "requests-mock~=1.12.1",
```

Comment:
Do we need it?

Reply:
yes, added in tests for the injector code that relies on mocking the requests library